transcendence-app/src/user/user.controller.spec.ts

```diff
@@ -1,31 +1,29 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { AvatarFileInterceptor, UserController } from './user.controller';
 import { UserService } from './user.service';
-import { CreateUserRequestDto } from './dto/user.dto';
+import { CreateUserDto } from './dto/create-user.dto';
```

Comment:
We could have a denomination for incoming/outcoming dtos. I was thinking in something like `*RequestDto`, `*ResponseDto` you've deleted :sweat_smile: Why don't you like it?

Reply:
Discussed by call: this will apply for Response only since, if done for requests also, we'd have many changes in webapp also